hisat2.2.1/1.0.0/unittest.yml

```diff
@@ -2,4 +2,34 @@ commands:
   - cmd: "hisat2 --version | head -n 2"
     expect_text: |
       /usr/bin/hisat2-2.2.1/hisat2-align-s version 2.2.1
-      64-bit
\ No newline at end of file
+      64-bit
+  - cmd: "gtfToGenePred 2>&1 | sed -e '/^$/d; s/^[ \t]*//; s/[ \t]*$//'"
```

Comment:
why do you need all of this `2>&1 | sed -e '/^$/d; s/^[ \t]*//; s/[ \t]*$//'`?

Reply:
Parse the standard error?

Reply:
why though? why not just run `gtfToGenePred` the output is the output... just like you don't need to do all of that for the Hisat2 test....

Reply:
The output is from STDERR and I think the parsing to see if its compatiable requires some string replacement.

But I think a head could be sufficient.

Reply:
So, for whatever reason, the pytest script only sees the STDOUT, and not STDERR, so that is why the redirect.  The other `sed `statement is there to function similar to Pearl's chomp, because I was getting weird spacing in the front and end of lines, which the yaml wouldn't pick up.

Reply:
Changing to just `head` would work, too.